elfi/examples/gauss_nd.py

```diff
@@ -0,0 +1,130 @@
+"""The general, n-dimensional Gaussian noise model."""
+
+from functools import partial
+
+import numpy as np
+import scipy.stats as ss
+
+import elfi
+from elfi.examples.gnk import euclidean_multidim
+
+
+def Gauss_nd(*params, cov_ii=None, cov_ij=None, n_obs=15, batch_size=1, random_state=None):
+    """Sample the Gaussian distribution.
+
+    Reference
+    ---------
+    The default settings replicate the experiment settings used in [1].
+
+    [1] arXiv:1704.00520 (Järvenpää et al., 2017).
+
+    Parameters
+    ----------
+    *params : array_like
+        The array elements correspond to the mean parameters.
+    cov_ii : float, optional
+        The diagonal variance.
+    cov_ij : float, optional
+        The non-diagonal variance.
```

Comment:
Since these are now constant, I think it would be both simpler and more flexible to require the cov matrix as input. A lot of the code below could then be removed.